core/messagesource.go

```diff
@@ -22,6 +22,9 @@ type MessageSource interface {
 
 	// IsBlocked returns true if the source cannot produce messages
 	IsBlocked() bool
+
+	// GetID returns the pluginID of the message source
+	GetID() string
```

Comment:
how discussed: because of the reason that formatters are plugins and have no id's we let this change that it is for now.